lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -17,9 +18,12 @@ describe('BasicLtiLaunchApp', () => {
   let fakeRpcServer;
 
   const renderLtiLaunchApp = (props = {}) => {
+    const services = new Map([[ClientRpc, fakeRpcServer]]);
     return mount(
       <Config.Provider value={fakeConfig}>
-        <BasicLtiLaunchApp clientRpc={fakeRpcServer} {...props} />
+        <Services.Provider value={services}>
```

Comment:
This test doesn't use the `useServices` wrapper. Is this just a quicker/alternate way to do it?

Reply:
You mean the `withServices` wrapper? In this context I found more convenient not to use it because we also need the `Config.Provider` wrapper.